deps/childprocess.lua

```diff
@@ -164,4 +164,71 @@ local function spawn(command, args, options)
   return em
 end
 
+local function exec(command, args, options, callback)
+  local file
+  local stringArgs = command..table.concat(args, ' ')
+  if process.env._system_name:lower() == 'win32' then
```

Comment:
Also new luvi has `require('luvipath').isWindows` as a boolean.  This uses `jit.os == 'Windows'` on luajit and uses `not not package.path:match("\\")` as a fallback on other luas.